source/ui/IDEAS/IDEASXASScanConfigurationView.cpp

```diff
@@ -398,3 +420,17 @@ void IDEASXASScanConfigurationView::onFluorescenceChoiceChanged(int id)
 {
 	configuration_->setFluorescenceDetector(id);
 }
+
+void IDEASXASScanConfigurationView::onPeakingTimeChanged(double value)
```

Comment:
Ah, I see.  Unfortunately, we need to be a little careful.  We want to minimize the amount of beamline calls inside of scan configurations and configuration views.  We can have a chat about that sometime as well if you'd like.